get-to-know-security-manager/onedrive-screen.md

```diff
@@ -0,0 +1,11 @@
+---
+title: OneDrive Screen
+description: Detailed overview of connected OneDrive accounts.
+author: Petra Filipi
+date: 02/10/2018
+---
+The OneDrive screen gives you a general overview of your connected personal OneDrive accounts, as well as a report for each OneDrive account.
+
+### Available actions 
+* __Disconnect OneDrive__ – This action will remove permissions in Office 365 and connections in SysKit Security Manager so you will no longer be the owner of any OneDrive in the selected tenant.
```

Comment:
ovdje se malo implicira kao da vise nigdje neces biti owner, ali sebi ces naravno ostati. Takodjer, removeanje prava je opcionalno, pa mozda nesto u stilu:
"This action will remove connected OneDrives from SysKit Security Manager. Optionally you can remove added administrator privileges on all users OneDrives except your personal one."

Reply:
kako je u aplikaciji tako je i na helpu, neću zasad mijenjati.